app/helpers/appointment_helper.rb

```diff
@@ -2,4 +2,24 @@ module AppointmentHelper
   def friendly_options(statuses)
     statuses.map { |k, _| [k.titleize, k] }.to_h
   end
+
+  def rebooked_from(appointment)
+    return unless appointment.rebooked_from_id?
+
+    content_tag(
+      :span,
+      "rebooked from ##{appointment.rebooked_from_id}",
```

Comment:
Not a big deal but would be nice for this to be a link back to this appointment maybe?